app/assets/stylesheets/main.css.scss

```diff
@@ -1,17 +1,19 @@
-@import 'compass';
-@import 'font-awesome';
-@import 'styles/variables'; // colors
-@import 'styles/foundation_and_overrides';
-@import 'styles/presets';
-@import 'styles/devise';
-@import 'styles/components/nav';
-@import 'styles/ui/icons';
-@import 'styles/alerts';
-@import 'styles/components/crop_searches/**/*';
-@import 'styles/components/crops/**/*';
-@import 'styles/components/guides/**/*';
-@import 'styles/components/users/**/*';
-@import 'styles/ui/compatibility_circle';
+@import "compass";
+@import "font-awesome";
+@import "styles/variables"; // colors
+@import "styles/foundation_and_overrides";
+@import "styles/presets";
+@import "styles/devise";
+@import "styles/components/nav";
+@import "styles/ui/icons";
+@import "styles/alerts";
+@import "styles/components/crop_searches/**/*";
+@import "styles/components/crops/**/*";
+@import "styles/components/guides/**/*";
+@import "styles/components/users/**/*";
+@import "styles/ui/compatibility_circle";
```

Comment:
Haha well, once Hound asked me to do one, I figured I would do em all for consistency's sake